### ISSUE TITLE
Do you recommend pruning SNPs that are in linkage disequilibrium before putting into varbvs?

### ISSUE THREAD

**Issue by garyzhubc** [pcarbo/varbvs#34]
Do you recommend pruning SNPs that are in linkage disequilibrium before feeding into varbvs? It looks like paper suggests that the variational approximation works better under independence of the SNPs?

**Comment by pcarbo** [comment]
> The problem with pruning is that it may prune away the effective SNPs while leaving the ones that are merely in linkage disequilibrium with the effective ones.

Precisely. So if that is a concern, you should not prune SNPs.

**Comment by garyzhubc** [comment]
But would that come at a cost that varbvs approximation become inaccurate?

**Comment by pcarbo** [comment]
Yes, although despite this inaccuracy, some posterior statistics remain accurate, such as aggregate statistics over all SNPs in a chromosomal region; see the *Bayesian Analysis* paper for more discussion on this.
